core/src/components/toast/toast.tsx

```diff
@@ -71,6 +71,9 @@ export class Toast implements ComponentInterface, OverlayInterface {
   private customHTMLEnabled = config.get('innerHTMLTemplatesEnabled', ENABLE_HTML_CONTENT_DEFAULT);
   private durationTimeout?: ReturnType<typeof setTimeout>;
   private gesture?: Gesture;
+  private get theme(): Theme {
+    return getIonTheme(this);
+  }
```

Comment:
Let's remove this and just use `getIonTheme` within `getShape` because the structure would be consistent and because this function is redundant to `getIonTheme`.

```suggestion
```

Reply:
The reason for this method is because I was concerned that the theme could change in runtime (hence I did not create a simple field) and to avoid code repetition. However, I get your feedback and will follow your suggestion.